### PR TITLE
retry on error for failed tests

### PR DESCRIPTION
**Pull request by grzanka**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@ jobs:
               with:
                 resource: http://localhost:3000
             - run: npm test
+              retry_on: error
+              max_attempts: 2
+              timeout_seconds: 5
```
Comment (Bogdan2423) [inline .github/workflows/node.js.yml:44]: It shows this error when you go into checks:
error
I dont know why it show that all checks have passed- it looks like the node checks have been skipped in pull request.
According to this: https://www.thisdot.co/blog/how-to-retry-failed-steps-in-github-action-workflows/, I think this fragment should look like this:
```suggestion
            - name: Run tests 
              uses: nick-fields/retry@v2
	      with:
                max_attempts: 2
                retry_on: error
                timeout_seconds: 5
                command: npm test	
```

**Review by Bogdan2423 — CHANGES_REQUESTED** [review]
It shows this error when you go into checks:
error
I dont know why it show that all checks have passed- it looks like the node checks have been skipped in pull request.
According to this: https://www.thisdot.co/blog/how-to-retry-failed-steps-in-github-action-workflows/, I think it should look like

**Review by Bogdan2423 — CHANGES_REQUESTED** [review]
now it shows this error:
error2
error1
when I go to edit mode on the node.js.yml file it shows this:
error3
- maybe this is the problem? Still don't know why not all checks trigger and it shows that all checks have passed

**Comment by grzanka** [comment]
> now it shows this error: error2 error1 when I go to edit mode on the node.js.yml file it shows this: error3
> 
> * maybe this is the problem? Still don't know why not all checks trigger and it shows that all checks have passed

Yes, stupid mistake in indents, should work now

**Comment by Bogdan2423** [comment]
looks like all the checks launch now. However:
timeout
Looks like you need to set the timeout for enough time for all the tests to pass - that would be at least one minute, or more to be safe

**Comment by grzanka** [comment]
This won't work with `nick-fields/retry@v2` as it calls testing process from within a docker instance. This won't have access to the app instance served by `npm runs start` due to docker isolation environment

**Review by Bogdan2423 — APPROVED** [review]
lgtm

**Pull request merged**